tests/integration-tests/tests/schedulers/test_slurm.py

```diff
@@ -284,6 +285,31 @@ def test_slurm_protected_mode(
     _test_recover_from_protected_mode(pending_job_id, pcluster_config_reader, bucket_name, cluster, scheduler_commands)
 
 
+@pytest.mark.usefixtures("region", "os", "instance", "scheduler")
+@pytest.mark.fast_capacity_failover
+def test_fast_capacity_failover(
+    pcluster_config_reader,
+    clusters_factory,
+    test_datadir,
+    scheduler_commands_factory,
+):
+    cluster_config = pcluster_config_reader()
+    cluster = clusters_factory(cluster_config)
+    remote_command_executor = RemoteCommandExecutor(cluster)
+    clustermgtd_conf_path = _retrieve_clustermgtd_conf_path(remote_command_executor)
+    scheduler_commands = scheduler_commands_factory(remote_command_executor)
+    # after the cluster is launched, apply the override patch to launch ice nodes
+    remote_command_executor.run_remote_script(str(test_datadir / "overrides.sh"), run_as_root=True)
+    # test disable ice logic
+    job_id = _test_disable_fast_capacity_failover(scheduler_commands, remote_command_executor, clustermgtd_conf_path)
+    # test enable fast instance capacity failover
+    _test_enable_fast_capacity_failover(scheduler_commands, remote_command_executor, job_id, clustermgtd_conf_path)
+    # test update without queue param change, clustermgtd and slurmctld not restart
+    _test_update_without_update_queue_params(pcluster_config_reader, cluster, remote_command_executor)
+    # test update with queue param change, clustermgtd and slurmctld restart
+    _test_update_with_queue_params(pcluster_config_reader, cluster, remote_command_executor)
```

Comment:
I'd probably move these 2 in a separate test named something like test_slurm_config_update

Reply:
Added a separate test